src/Integration.UnitTests/Service/SonarQubeServiceWrapperTests.cs

```diff
@@ -155,6 +155,47 @@ public void SonarQubeServiceWrapper_TryGetProjects_Organization()
         }
 
         [TestMethod]
+        public void SonarQubeServiceWrapper_TryGetNotifications()
+        {
+            var expected = new NotificationEvent[]
+            {
+                new NotificationEvent { Category = "QUALITY_GATE",
+                    Message =  "Quality Gate of project 'test' is now Red (was Green)",
+                    Link = new Uri("http://localhost:9000/dashboard?id=test"),
+                    Date = new DateTimeOffset(2017, 1, 1, 9, 55, 1, 0, TimeSpan.FromHours(1)),
+                    Project = "test" }
+            };
+
+            using (var testSubject = new TestableSonarQubeServiceWrapper(this.serviceProvider))
+            {
+                testSubject.RegisterQueryValidator("api/developers/search_events", request =>
+                {
+                    request.QueryString.HasValue.Should().BeTrue();
+                    request.QueryString.Value.Should().Be("projects=test&from=2017-09-14T10%3a00%3a00%2b0200");
+                });
+
+                // Arrange
+                testSubject.AllowAnonymous = true;
+                testSubject.RegisterRequestHandler("api/developers/search_events?projects=test&from=2017-01-01T07%3a55%3a01%2b0200",
+                    new RequestHandler { ResponseText = Serialize(new { events = expected }) });
+
+                var connectionInfo = new ConnectionInformation(new Uri("http://server"));
+                var projectKey = "test";
+                var dateLastCheck = new DateTimeOffset(2017, 1, 1, 7, 55, 1, 0, TimeSpan.FromHours(2));
+
+                // Act
+                NotificationEvent[] events;
+
+                testSubject.TryGetNotificationEvents(connectionInfo, CancellationToken.None, projectKey, dateLastCheck,
```

Comment:
I would store the result into a variable and make the should be true test in the 'Assert' section.